keycloak/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM jboss/keycloak-postgres:3.2.1.Final
+FROM jboss/keycloak-postgres:3.3.0.Final
```

Comment:
Let's not bump the keycloak version up just yet.  There's likely more work to do there, and the `keycloak-postgres` image is [now deprecated](https://hub.docker.com/r/jboss/keycloak-postgres/) anyway.